### DISCUSSION TITLE
streamtape no longer downloads

### DISCUSSION THREAD

**Discussion by meatyguitar** [aclap-dev/video-downloadhelper#601] [Help and Q&A]
### Your question

Streamtape download failed. Worked fine before in the previous version 

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://tapeadvertisement.com/v/wdaXk1VRXzFJq7y/Civil.War.2024.1080p.AMZN.WEBRip.1400MB.DD5.1.x264-GalaxyRG.mp4",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 00000253ff1d79c0] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://861481081.tapecontent.net/radosgw/wdaXk1VRXzFJq7y/rlGuHF15ock8z5JDcmtff_i-7ZmeBDkzJZXmIbWdQttYjt-TXCouJfqPOm7iRgD-rCFj8IYY9FeZN7jgNS0S6WiB6FUT6HxoWkFAZ2yze9kaTnXYHZhxhCQGjXIQvin9Aq7drDA-qq_Z4EnE9mMRv3LLg1-l_VfKxeZygHqmfwlNtZnD9XZLHQX82Ec-Whtz1quD1dq7lE2vv8-7fqTw256EbBaSk8rUXueq3-uNAZvDsuVfqbhtRir7aIwEkRG17oLXfGDH1lBvUVh4hab5JOe0FvJ5dle-iM9S4WybV3sjmO-DEjgu5cCcGfpqEOiJo_brgMbpmaSMJ9Ke/Civil.War.2024.1080p.AMZN.WEBRip.1400MB.DD5.1.x264-GalaxyRG.mp4?stream=1.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "id": "coapp_failure_33e7713c-4ea2-4322-9705-38752fa86227",
    "downloadable_id": "downloadable_6630907104800037"
  }
}

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
{
    "download_directory": "D:\\[2024 DL]",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 22
}

**Comment by paulrouget** [comment]
If I press "play" first, I see another video, which seems to work:

<img width="427" alt="Screenshot 2024-05-24 at 15 15 33" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/6ed42a1e-1e34-4cbb-a9b3-c2f6db49d6ce">
